alternator/executor.cc

```diff
@@ -96,6 +96,44 @@ std::string make_jsonable::to_json() const {
     return rjson::print(_value);
 }
 
+
+static bool is_leaf(const rjson::value& value) {
+    return !value.IsObject() && !value.IsArray();
+}
+
+static future<> destroy_gently_nonleaf(rjson::value&& value_in);
+
+future<> destroy_gently(rjson::value&& value) {
+    if (is_leaf(value)) {
+        return make_ready_future<>();
+    } else {
+        return destroy_gently_nonleaf(std::move(value));
+    }
+}
+
+static
+future<>
+destroy_gently_nonleaf(rjson::value&& value_in) {
```

Comment:
Coding style nitpick: please put the "static future<>" on the same line as the function name, like other functions in this file and also the other functions you defined in this patch.

Reply:
ok